android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java

```diff
@@ -248,15 +253,23 @@ public void onDownloadStart(String url, String userAgent, String contentDisposit
 
         module.setDownloadRequest(request);
 
-        if (module.grantFileDownloaderPermissions()) {
-          module.downloadFile();
+        if (module.grantFileDownloaderPermissions(getDownloadingMessage(), getLackPermissionToDownloadMessage())) {
+          module.downloadFile(getDownloadingMessage());
         }
       }
     });
 
     return webView;
   }
 
+  private String getDownloadingMessage(){
```

Comment:
ultra-nit, but it would be good to follow the style standard of the rest of the file, which would mean adding a space before the bracket

(as well as updating the setters so that the opening bracket is on the same line as the function signature)

(also it's slightly weird to have the getters separated from the setters, but I totally get that to do otherwise would break the grouping of React props, so I don't have a better idea, just noting that quirk)

Reply:
Fixed the spacing and placement of the function brackets